Services/CAS/classes/class.ilCASSettingsGUI.php

```diff
@@ -87,7 +92,14 @@ protected function initFormSettings(): ilPropertyFormGUI
         $form->setFormAction($this->ctrl->getFormAction($this));
 
         $form->setTitle($this->lng->txt('auth_cas_auth'));
-        $form->setDescription($this->lng->txt("auth_cas_auth_desc"));
+
+        $dropInReplacementsUrl = 'https://github.com/ILIAS-eLearning/ILIAS/tree/release_8/src/HTTP#dropinreplacements';
+        $dropInReplacementsLink = $this->factory->link()->standard($dropInReplacementsUrl, $dropInReplacementsUrl);
+        $form->setDescription(sprintf(
+            $this->lng->txt("auth_cas_auth_desc"),
+            '7.1.0',
+            $this->renderer->render($dropInReplacementsLink)
+        ));
```

Comment:
@mjansenDatabay The `\ilSetupAgent::PHP_MIN_VERSION` is private. I guess we can drop the text about the requirements about PHP and CURL altogether because composer takes care of that.